sdk/arm/resources/ci.yml

```diff
@@ -0,0 +1,15 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+trigger:
+  paths:
+    include:
+    - sdk/arm/resources/
+
+pr:
+  paths:
+    include:
+    - sdk/arm/resources/
```

Comment:
Should this just be arm? Or will we be adding each module path here?

Reply:
Each RP will have its own pipeline so that we don't have to build and verify the world when a subset changes (keep in mind there will be a lot of API versions per RP so it will add up).

Reply:
Ok makes sense 👍